frontend/src/util/i18n.js

```diff
@@ -70,23 +70,12 @@ const kanbanPtTranslation = require('../../dist/app/kanban_pt_translation.json')
 const kanbanDeTranslation = require('../../dist/app/kanban_de_translation.json')
 const kanbanArTranslation = require('../../dist/app/kanban_ar_translation.json')
 
-export const getBrowserLang = () => {
-  const browserLang = navigator.language
-
-  if (['en', 'fr', 'pt', 'de', 'ar'].includes(browserLang)) return browserLang
-  if (browserLang.includes('fr')) return 'fr' // for fr-XX
-  if (browserLang.includes('pt')) return 'pt' // for pt-XX
-  if (browserLang.includes('de')) return 'de' // for de-XX
-  if (browserLang.includes('ar')) return 'ar' // for de-XX
-
-  return 'en'
-}
-
 i18n
   .use(reactI18nextModule)
   .init({
     fallbackLng: getBrowserLang(),
     // have a common namespace used around the full app
+    returnEmptyString: false,
```

Comment:
What are these `returnEmptyString: false` for?

Reply:
If true, allows empty string as valid translation. Setting as false instead of an empty translation, it uses the fallback language.